### ISSUE TITLE
Help me

### ISSUE THREAD

**Issue by IndogLelele** [GrowtopiaNoobs/GrowtopiaServer#1027]
Help me i want make a like a file will auto update every a second

**Comment by Bykosw** [comment]
**1. #include <thread>
2. long long int update = 0;
3. void sss()
{
	while (sss)
	{
		using namespace std::chrono;
		if (update + 1000 < (duration_cast<milliseconds>(system_clock::now().time_since_epoch())).count())
		{
			update = (duration_cast<milliseconds>(system_clock::now().time_since_epoch())).count();
			cout << "." << endl;
		}
	}
}
4. std::thread upd(sss);
    upd.detach();**

**Comment by IndogLelele** [comment]
@Bykosw Sorry i did make it xD
